__tests__/index.js

```diff
@@ -54,3 +55,17 @@ test('works', () => {
       ])
     })
 })
+
+test('terminates', () => {
+  const range$ = fromIter([10, 20, 30, 40])
+  const source$ = remember(range$)
+
+  let actual = []
+
+  pipe(
+    source$,
+    forEach(i => actual.push(i)),
```

Comment:
I feel like we should also check that we got `2` correctly in this case, could be achieved with i.e. [`callbag-tap`](https://github.com/krawaller/callbag-tap/blob/031c54a6337d9587ce3b9be20b2b5791aa8ca1e3/index.js#L7) or you could leverare [`callbag-mock`](https://github.com/krawaller/callbag-mock) for the whole test, WDYT?

Reply:
Aggred! :) I updated the PR

Reply:
Not sure which one I actually prefer at the moment - I'd like to write tests as close to how package is used as possible and kinda didn't yet found my preferred way of doing that with callbags - `callbag-tap` is fine for me, it gets job done 😉